plugin/pkg/admission/securitycontext/scdeny/admission.go

```diff
@@ -79,21 +79,21 @@ func (p *Plugin) Validate(ctx context.Context, a admission.Attributes, o admissi
 	for _, v := range pod.Spec.InitContainers {
 		if v.SecurityContext != nil {
 			if v.SecurityContext.SELinuxOptions != nil {
-				return apierrors.NewForbidden(a.GetResource().GroupResource(), pod.Name, fmt.Errorf("SecurityContext.SELinuxOptions is forbidden"))
```

Comment:
These four occurrences I don't know. They are referring to a field that is capitalized, I would probably leave them as-is